Deploy/python3/check-api-status.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python3
+
+import json
+import requests
+
+
+def lambda_handler(event, context):
+    # print statement for local testing (not deployed to lambda)
+    print(event + context)
+    status = "Up"
+    health_response = requests.get("https://api.ab2d.cms.gov/health")
+    if health_response.status_code != 200:
+        status = "Down"
+    status_response = requests.get("https://api.ab2d.cms.gov/status")
+    if status_response.status_code != 200:
+        status = "Down"
+    status_response_json = json.loads(status_response.text)
+    if status_response_json['maintenanceMode'] == 'true':
```

Comment:
So there was a question here about whether this should be included for now or not. Because we can still accept jobs in maintenance mode. We just can't finish them.

Reply:
The pros and cons were discussed, but ultimately Laila said to include this check.